src/Compilers/VisualBasic/Test/CommandLine/ErrorLoggerTests.vb

```diff
@@ -115,6 +115,7 @@ End Class
         ""title"": ""Unused local variable"",
         ""category"": ""Compiler"",
         ""isEnabledByDefault"": ""True"",
+        ""isSuppressedInSource"": ""False"",
```

Comment:
the value emitted here is as per the SARIF spec: https://github.com/sarif-standard/sarif-spec.